tests/foreman/api/test_repositories.py

```diff
@@ -190,3 +193,28 @@ def test_positive_sync_kickstart_repo(self, module_entitlement_manifest_org, tar
     rh_repo = rh_repo.read()
     assert rh_repo.content_counts['package_group'] > 0
     assert rh_repo.content_counts['rpm'] > 0
+
+
+@pytest.mark.tier1
+def test_negative_upload_expired_manifest(module_org, target_sat):
+    """Upload an expired manifest and attempt to refresh it
```

Comment:
Do you think the case level and case importance is right here thats being fetched from module hierarchy?

Reply:
most tests prior have been marked to tier1 due to these being support cases. If you think I should remove this, should I remove for all other tests too ?